.github/workflows/reusable-build-and-push.yml

```diff
@@ -12,63 +12,132 @@ on:
         type: boolean
         description: 'If set to true, push the image.'
         default: false
+      stable_release:
+        required: false
+        type: boolean
+        description: 'If set to true the image is pushed to the stable repository otherwise pushed to the development.'
+        default: false
+      post_images:
+        required: false
+        type: boolean
+        description: |
+          If set to true the image tags pushed to the repository are posted as comment for the Pull Request.
+          Only works if the `stable_release` is set to false.
```

Comment:
> Only works if the `stable_release` is set to false

How this is being enforced?

Reply:
Valid. I change the logic to make sure posting image tags only works for PRs. I removed the restriction for the comments are only posted if the image push target is the development repository.